src/OrchardCore/OrchardCore/Shell/Configuration/ShellConfigurationSources.cs

```diff
@@ -87,5 +97,51 @@ public Task RemoveAsync(string tenant)
 
             return Task.CompletedTask;
         }
+
+        private static JToken ConfigToJObject(IConfiguration configuration)
```

Comment:
We might need to add this as extension method to `IConfiguration`, so could be used anywhere across the code-base

Reply:
Yes I will because it will be used in our other `IShellConfigurationSources` for database and blob.

Work in progress